Prepare docs for 0.13 release (#133)
kubeshop/botkube-docs: content/installation/Discord/_index.md

### Install BotKube Backend in Kubernetes cluster

> - We will be using [Helm](https://helm.sh/) to install BotKube in Kubernetes. Follow [this](https://docs.helm.sh/using_helm/#installing-helm) guide to install helm if you don't have it installed already.

Reviewer: ```suggestion
- We use [Helm](https://helm.sh/) to install BotKube in Kubernetes. Follow [this](https://docs.helm.sh/using_helm/#installing-helm) guide to install helm if you don't have it installed already.
```

Would be nice to apply this suggestion everywhere.

- Deploy BotKube backend using **helm install** in your cluster:

  ```bash
  $ helm install --version v0.13.0 botkube --namespace botkube --create-namespace \
  --set communications.discord.enabled=true \
  --set communications.discord.channel=<DISCORD_CHANNEL_ID> \
  --set communications.discord.botid=<DISCORD_BOT_ID> \
  --set communications.discord.token=<DISCORD_TOKEN> \

>   --set config.settings.clusterName=<CLUSTER_NAME> \

Author: I didn't touch Discord doc because your PR addresses all the required changes I think. Will double check once you merge yours
